### ISSUE TITLE
Shortcuts / apps on macOS throws exception 'NSInvalidArgumentException' instantly

### ISSUE THREAD

**Issue by mdbraber** [brave/brave-browser#7205]
## Description 
When creating an 'app' on macOS 10.15.1 it immediately throws the following error:

```
2019-12-04 17:30:20.202 app_mode_loader[32295:1717699] -[NSAccessibilityRemoteUIElement accessibilityTitle]: unrecognized selector sent to instance 0x7fe76de4e7d0
2019-12-04 17:30:20.204 app_mode_loader[32295:1717699] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[NSAccessibilityRemoteUIElement accessibilityTitle]: unrecognized selector sent to instance 0x7fe76de4e7d0'
```

## Steps to Reproduce

   1. Go to fastmail.com (or another site like community.brave.com) 
   2. Select More Tools > Create Shortcut and create the shortcut with 'Open as window' selected
   3. Launch the shortcut

## Actual result:
Briefly shows website in separate window, crashes right after. Shows the error above. Full error when launching `app_mode_loader` separately:

```
objc[32404]: Class BaseView is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083a98) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c472cf8). One of the two will be used. Which one is undefined.
objc[32404]: Class CommandDispatcher is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083ae8) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c472d48). One of the two will be used. Which one is undefined.
objc[32404]: Class ConstrainedWindowAnimationBase is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083b38) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c472d98). One of the two will be used. Which one is undefined.
objc[32404]: Class ConstrainedWindowAnimationShow is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083bb0) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c472e10). One of the two will be used. Which one is undefined.
objc[32404]: Class ConstrainedWindowAnimationHide is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083c00) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c472e60). One of the two will be used. Which one is undefined.
objc[32404]: Class ConstrainedWindowAnimationPulse is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083c50) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c472eb0). One of the two will be used. Which one is undefined.
objc[32404]: Class ButtonUtils is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083ca0) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c472f00). One of the two will be used. Which one is undefined.
objc[32404]: Class TextFieldUtils is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083cf0) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c472f50). One of the two will be used. Which one is undefined.
objc[32404]: Class FindPasteboard is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083d18) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c472f78). One of the two will be used. Which one is undefined.
objc[32404]: Class FlippedView is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083d90) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c472ff0). One of the two will be used. Which one is undefined.
objc[32404]: Class FocusTracker is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083db8) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c473018). One of the two will be used. Which one is undefined.
objc[32404]: Class MenuControllerCocoa is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083e08) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c473068). One of the two will be used. Which one is undefined.
objc[32404]: Class ResponsiveNSMenuItem is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083e30) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c473090). One of the two will be used. Which one is undefined.
objc[32404]: Class ToolTipBaseView is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083ea8) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c473108). One of the two will be used. Which one is undefined.
objc[32404]: Class CrTrackingAreaOwnerProxy is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083ef8) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c473158). One of the two will be used. Which one is undefined.
objc[32404]: Class CrTrackingArea is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083f48) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c4731a8). One of the two will be used. Which one is undefined.
objc[32404]: Class UnderlayOpenGLHostingWindow is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c083f98) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c4731f8). One of the two will be used. Which one is undefined.
objc[32404]: Class WeakPtrNSObject is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c084010) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c473270). One of the two will be used. Which one is undefined.
objc[32404]: Class CrPasteboardItemWrapper is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c084038) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c473298). One of the two will be used. Which one is undefined.
objc[32404]: Class NowPlayingInfoCenterDelegateCocoa is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c084088) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c4732e8). One of the two will be used. Which one is undefined.
objc[32404]: Class RemoteCommandCenterDelegateNS is implemented in both /Users/mdbraber/Applications/Brave Browser Apps.localized/Fastmail.app/Fastmail.app/Contents/MacOS/./app_mode_loader (0x10c0840d8) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/78.1.0.1/Brave Browser Framework (0x11c473338). One of the two will be used. Which one is undefined.
2019-12-04 17:37:53.041 app_mode_loader[32404:1724145] -[NSAccessibilityRemoteUIElement accessibilityTitle]: unrecognized selector sent to instance 0x7fedc9825aa0
2019-12-04 17:37:53.043 app_mode_loader[32404:1724145] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[NSAccessibilityRemoteUIElement accessibilityTitle]: unrecognized selector sent to instance 0x7fedc9825aa0'
*** First throw call stack:
(
	0   CoreFoundation                      0x00007fff2fdd2f53 __exceptionPreprocess + 250
	1   libobjc.A.dylib                     0x00007fff65e98835 objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff2fe5d106 -[NSObject(NSObject) __retain_OA] + 0
	3   CoreFoundation                      0x00007fff2fd796cb ___forwarding___ + 1427
	4   CoreFoundation                      0x00007fff2fd790a8 _CF_forwarding_prep_0 + 120
	5   Brave Browser Framework             0x000000011842c9b1 ChromeMain + 78342513
	6   AppKit                              0x00007fff2cff58e2 -[NSAccessibilityAttributeAccessorInfo getAttributeValue:forObject:] + 58
	7   AppKit                              0x00007fff2d57ef02 ___NSAccessibilityEntryPointValueForAttribute_block_invoke.805.llvm.7288867694539378565 + 1674
	8   AppKit                              0x00007fff2d57ac29 NSAccessibilityPerformEntryPointObject.llvm.7288867694539378565 + 16
	9   AppKit                              0x00007fff2d57ad6a _NSAccessibilityEntryPointValueForAttribute.llvm.7288867694539378565 + 168
	10  AppKit                              0x00007fff2d36c59a -[NSObject(NSAccessibilityInternal) _accessibilityValueForAttribute:clientError:] + 351
	11  AppKit                              0x00007fff2d371736 CopyAppKitUIElementAttributeValueNoCatch + 98
	12  AppKit                              0x00007fff2d36e619 CopyAttributeValue + 142
	13  HIServices                          0x00007fff2dfb0b9d _AXXMIGCopyAttributeValue + 389
	14  HIServices                          0x00007fff2dfd252b _XCopyAttributeValue + 443
	15  HIServices                          0x00007fff2df90553 mshMIGPerform + 205
	16  CoreFoundation                      0x00007fff2fd39d9f __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE1_PERFORM_FUNCTION__ + 41
	17  CoreFoundation                      0x00007fff2fd39cef __CFRunLoopDoSource1 + 541
	18  CoreFoundation                      0x00007fff2fd21c9c __CFRunLoopRun + 2612
	19  CoreFoundation                      0x00007fff2fd20fe3 CFRunLoopRunSpecific + 499
	20  HIToolbox                           0x00007fff2e8a867d RunCurrentEventLoopInMode + 292
	21  HIToolbox                           0x00007fff2e8a83bd ReceiveNextEventCommon + 600
	22  HIToolbox                           0x00007fff2e8a8147 _BlockUntilNextEventMatchingListInModeWithFilter + 64
	23  AppKit                              0x00007fff2cf2d864 _DPSNextEvent + 990
	24  AppKit                              0x00007fff2cf2c5d4 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1352
	25  AppKit                              0x00007fff2cf26d76 -[NSApplication run] + 658
	26  Brave Browser Framework             0x000000011687e5bc ChromeMain + 49317244
	27  Brave Browser Framework             0x000000011687d0e8 ChromeMain + 49311912
	28  Brave Browser Framework             0x00000001168cf408 ChromeMain + 49648584
	29  Brave Browser Framework             0x00000001168a3137 ChromeMain + 49467639
	30  Brave Browser Framework             0x0000000113975cae ChromeAppModeStart_v6 + 2094
	31  app_mode_loader                     0x000000010bb9d8d1 main + 2449
	32  app_mode_loader                     0x000000010bb9cf5b main + 27
	33  libdyld.dylib                       0x00007fff671fb2e5 start + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
[1204/173753.182120:WARNING:system_snapshot_mac.cc(42)] sysctlbyname kern.nx: No such file or directory (2)
```

## Expected result:
Display a separate window with the specific website and not crash ;-)

## Reproduces how often: 
Always reproducible

## Brave version (brave://version info)
Brave | 1.0.1 Chromium: 78.0.3904.108 (Official Build) (64-bit)
-- | --
Revision | 4b26898a39ee037623a72fcfb77279fce0e7d648-refs/branch-heads/3904@{#889}
OS | macOS Version 10.15.1 (Build 19B88)
JavaScript | V8 7.8.279.23
Flash | (Disabled)
User Agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36
Command Line | /Applications/Brave Browser.app/Contents/MacOS/Brave Browser --enable-dom-distiller --disable-domain-reliability --no-pings --extension-content-verification=enforce_strict --extensions-install-verification=enforce --enable-oop-rasterization=Enabled --sync-url=https://no-thanks.invalid --enable-features=NewExtensionUpdaterService,PasswordImport,WebUIDarkMode,SimplifyHttpsIndicator --disable-features=SmsReceiver,AudioServiceOutOfProcess,NotificationTriggers,AutofillServerCommunication,LookalikeUrlNavigationSuggestionsUI,UnifiedConsent,SyncUSSBookmarks --flag-switches-begin --flag-switches-end
Executable Path | /Applications/Brave Browser.app/Contents/MacOS/Brave Browser
Profile Path | /Users/mdbraber/Library/Application Support/BraveSoftware/Brave-Browser/Default

## Version/Channel Information:
- Can you reproduce this issue with the current release?
Yes

- Can you reproduce this issue with the beta channel?
Yes

- Can you reproduce this issue with the dev channel?
Did not try

- Can you reproduce this issue with the nightly channel?
Did not try

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
No

- Does the issue resolve itself when disabling Brave Rewards?
No

- Is the issue reproducible on the latest version of Chrome? 
No - tried latest version of Chromium (Version 77.0.3865.90 (Official Build) (64-bit)) and this works fine

## Similar reports
This issue has already been reported in the community forums thrice by others:

- https://community.brave.com/t/brave-browser-beta-apps-app-mode-launcher-throws-nsinvalidargumentexception-instantly-on-latest-macos-catalina-beta/77034
- https://community.brave.com/t/drag-problem-with-website-shortcuts-as-app/83316
- https://community.brave.com/t/brave-browser-apps-fail-to-open/128797/2

**Comment by mdbraber** [comment]
After some digging I've found that BetterSnapTool (https://folivora.ai/bettersnaptool) seems to be causing the problems. I'll try and see if I can find what exactly is causing this and why it's actually causing Brave to crash (and not BetterSnapTool)

**Comment by mdbraber** [comment]
Any pointers from the Brave developer what could be causing this is welcome (it would seem that Brave should still catch the thrown exception) @bsclifton

**Comment by mdbraber** [comment]
Did some more digging and it seems this is a bug in Brave, rather than BetterSnapTool. The `accessibilityTitle` property called should exist. I'm not sure what patches Brave uses that change this, because it does exist in the original Chrome source code (and e.g. Chromium doesn't crash from the same problem): https://github.com/chromium/chromium/search?q=accessibilityTitle&unscoped_q=accessibilityTitle

I've asked the BetterSnapTool developer about his knowledge about this particular type of crash: _"Maybe Brave uses a different kind of custom window for more customisability or something like that. In that case they should either disable Accessibility support completely for that window or implement all the required methods (or at least add stubs so it won't crash when macOS calls them)."_ (https://community.folivora.ai/t/bettersnaptool-and-bettertouchtool-cause-brave-web-browser-apps-to-crash/12299/5)

@bsclifton would you be able to elaborate? Also, it seems this should be tagged as a bug rather than a feature?

**Comment by bsclifton** [comment]
I'm curious if this is the same problem as https://github.com/brave/brave-browser/issues/7546

@mdbraber do you use the digital keyboard?

**Comment by mdbraber** [comment]
@bsclifton no, I don't use the digital keyboard, but BetterSnapTool does use many of the accessibility features, so I could be calling the same methods which make Brave crash

**Comment by digitalmaster** [comment]
I can confirm that this is the reason for my instant crashes too. Launched app shortcuts created with Brave causes the app to crash after any user interaction (ex, clicking on toolbar). Closing BetterSnapTool solves this issue.

I can also confirm that this doesn't happen for shortcut apps created by Chrome - just Brave app shortcuts.

**Comment by Brave-Matt** [comment]
+1 from Community user:
https://community.brave.com/t/brave-browser-apps-fail-to-open/128797/2

**Comment by mdbraber** [comment]
@Brave-Matt great if you could get some developer eyes on this!

**Comment by melMass** [comment]
Same issue here with Better Touch Tool :(

**Comment by digitalmaster** [comment]
I've had to kill the process. I miss my shortcuts 😢... I cry a little bit everytime I have to reach for my mouse to resize a window 😭😆

**Comment by bsclifton** [comment]
@mdbraber @melMass @digitalmaster can you all please try the Nightly build of Brave?
https://brave.com/download-nightly/

Try to create a shortcut and see if you run into the same problem. There's a good chance (with Chromium 83, which is on master and will be on Beta soon) it will be fixed

If that does fix it, we have a release coming next week 😄

**Comment by mdbraber** [comment]
@bsclifton thanks for picking this up! Something has changed, but unfortunately for the worse: Nightly unfortunately instantly crashes now instead of after moving a window. Same error it seems: `2020-06-05 10:39:46.653 app_mode_loader[84717:2118134] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[NSAccessibilityRemoteUIElement accessibilityTitle]: unrecognized selector sent to instance 0x7fb996c0a1e0'`


Here's the stack trace:

```
mdbraber-mbp:MacOS mdbraber$ ./app_mode_loader
[0605/103945.986324:INFO:app_mode_loader_mac.mm(76)] Using user data dir /Users/mdbraber/Library/Application Support/BraveSoftware/Brave-Browser-Nightly
[0605/103945.987713:INFO:app_mode_loader_mac.mm(95)] Framework version from symlink 83.1.11.50
[0605/103945.988575:INFO:app_mode_loader_mac.mm(134)] Loading framework /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework
objc[84717]: Class BaseView is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada94a0) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62ec30). One of the two will be used. Which one is undefined.
objc[84717]: Class CommandDispatcher is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada94f0) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62ec80). One of the two will be used. Which one is undefined.
objc[84717]: Class ConstrainedWindowAnimationBase is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada9540) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62ecd0). One of the two will be used. Which one is undefined.
objc[84717]: Class ConstrainedWindowAnimationShow is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada95b8) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62ed48). One of the two will be used. Which one is undefined.
objc[84717]: Class ConstrainedWindowAnimationHide is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada9608) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62ed98). One of the two will be used. Which one is undefined.
objc[84717]: Class ConstrainedWindowAnimationPulse is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada9658) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62ede8). One of the two will be used. Which one is undefined.
objc[84717]: Class ButtonUtils is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada96a8) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62ee38). One of the two will be used. Which one is undefined.
objc[84717]: Class TextFieldUtils is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada96f8) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62ee88). One of the two will be used. Which one is undefined.
objc[84717]: Class FindPasteboard is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada9720) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62eeb0). One of the two will be used. Which one is undefined.
objc[84717]: Class FlippedView is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada9798) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62ef28). One of the two will be used. Which one is undefined.
objc[84717]: Class FocusTracker is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada97c0) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62ef50). One of the two will be used. Which one is undefined.
objc[84717]: Class WeakPtrToMenuModelAsNSObject is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada9810) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62efa0). One of the two will be used. Which one is undefined.
objc[84717]: Class MenuControllerCocoa is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada9860) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62eff0). One of the two will be used. Which one is undefined.
objc[84717]: Class ToolTipBaseView is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada98b0) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62f040). One of the two will be used. Which one is undefined.
objc[84717]: Class CrTrackingAreaOwnerProxy is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada9900) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62f090). One of the two will be used. Which one is undefined.
objc[84717]: Class CrTrackingArea is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada9950) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62f0e0). One of the two will be used. Which one is undefined.
objc[84717]: Class UnderlayOpenGLHostingWindow is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada99a0) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62f130). One of the two will be used. Which one is undefined.
objc[84717]: Class WeakPtrNSObject is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada9a18) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62f1a8). One of the two will be used. Which one is undefined.
objc[84717]: Class CrPasteboardItemWrapper is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada9a40) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62f1d0). One of the two will be used. Which one is undefined.
objc[84717]: Class NowPlayingInfoCenterDelegateCocoa is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada9a90) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62f220). One of the two will be used. Which one is undefined.
objc[84717]: Class RemoteCommandCenterDelegateCocoa is implemented in both /Users/mdbraber/Applications/Brave Browser Nightly Apps.localized/NOS.app/Contents/MacOS/./app_mode_loader (0x10ada9ae0) and /Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/83.1.11.50/Brave Browser Nightly Framework (0x11e62f270). One of the two will be used. Which one is undefined.
[0605/103946.179627:INFO:app_shim_controller.mm(248)] Connected to com.brave.Browser.nightly.apps.027f39e5d2729fbfc2628e237b3d100b
[0605/103946.187376:INFO:app_shim_controller.mm(334)] Sent OnShimConnected
[0605/103946.242533:INFO:app_shim_controller.mm(366)] Received OnShimConnected.
2020-06-05 10:39:46.647 app_mode_loader[84717:2118134] -[NSAccessibilityRemoteUIElement accessibilityTitle]: unrecognized selector sent to instance 0x7fb996c0a1e0
2020-06-05 10:39:46.653 app_mode_loader[84717:2118134] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[NSAccessibilityRemoteUIElement accessibilityTitle]: unrecognized selector sent to instance 0x7fb996c0a1e0'
*** First throw call stack:
(
	0   CoreFoundation                      0x00007fff33781be7 __exceptionPreprocess + 250
	1   libobjc.A.dylib                     0x00007fff6c5595bf objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff33800c77 -[NSObject(NSObject) __retain_OA] + 0
	3   CoreFoundation                      0x00007fff336e644b ___forwarding___ + 1427
	4   CoreFoundation                      0x00007fff336e5e28 _CF_forwarding_prep_0 + 120
	5   Brave Browser Nightly Framework     0x000000011a36e971 ChromeMain + 82817217
	6   AppKit                              0x00007fff30a28a4a -[NSAccessibilityAttributeAccessorInfo getAttributeValue:forObject:] + 58
	7   AppKit                              0x00007fff30fe63f9 ___NSAccessibilityEntryPointValueForAttribute_block_invoke.805 + 1674
	8   AppKit                              0x00007fff30fe2244 NSAccessibilityPerformEntryPointObject + 16
	9   AppKit                              0x00007fff30ca63e1 _NSAccessibilityEntryPointValueForAttribute + 168
	10  AppKit                              0x00007fff30de7883 -[NSObject(NSAccessibilityInternal) _accessibilityValueForAttribute:clientError:] + 351
	11  AppKit                              0x00007fff30dec99e CopyAppKitUIElementAttributeValueNoCatch + 98
	12  AppKit                              0x00007fff30de9882 CopyAttributeValue + 142
	13  HIServices                          0x00007fff319dc3f6 _AXXMIGCopyAttributeValue + 389
	14  HIServices                          0x00007fff319fd4bb _XCopyAttributeValue + 443
	15  HIServices                          0x00007fff319bcf64 mshMIGPerform + 174
	16  CoreFoundation                      0x00007fff33706394 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE1_PERFORM_FUNCTION__ + 41
	17  CoreFoundation                      0x00007fff337062e0 __CFRunLoopDoSource1 + 541
	18  CoreFoundation                      0x00007fff33704e09 __CFRunLoopRun + 2270
	19  CoreFoundation                      0x00007fff33703ece CFRunLoopRunSpecific + 462
	20  HIToolbox                           0x00007fff32332abd RunCurrentEventLoopInMode + 292
	21  HIToolbox                           0x00007fff323327d5 ReceiveNextEventCommon + 584
	22  HIToolbox                           0x00007fff32332579 _BlockUntilNextEventMatchingListInModeWithFilter + 64
	23  AppKit                              0x00007fff3097a829 _DPSNextEvent + 883
	24  AppKit                              0x00007fff30979070 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1352
	25  AppKit                              0x00007fff3096ad7e -[NSApplication run] + 658
	26  Brave Browser Nightly Framework     0x0000000118824e4c ChromeMain + 54203804
	27  Brave Browser Nightly Framework     0x00000001188238b2 ChromeMain + 54198274
	28  Brave Browser Nightly Framework     0x00000001187cd1b3 ChromeMain + 53844227
	29  Brave Browser Nightly Framework     0x00000001187a3513 ChromeMain + 53673059
	30  Brave Browser Nightly Framework     0x0000000115473538 ChromeAppModeStart_v6 + 2296
	31  app_mode_loader                     0x000000010a8878f8 main + 2872
	32  app_mode_loader                     0x000000010a886ddb main + 27
	33  libdyld.dylib                       0x00007fff6d700cc9 start + 1
	34  ???                                 0x0000000000000001 0x0 + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
[0605/103946.880313:WARNING:system_snapshot_mac.cc(42)] sysctlbyname kern.nx: No such file or directory (2)
Abort trap: 6
```

**Comment by digitalmaster** [comment]
@bsclifton Thanks for looking into this. 

Exact same behavior for me. Crashes as soon as I perform some action (like trying to drag the window).

**Comment by digitalmaster** [comment]
Assuming `priority/P4` label means this won't be done any time soon?

**Comment by petemill** [comment]
I'm looking in to this, it's been affecting me this whole time

**Comment by halkazzar** [comment]
+1 - I do not use BetterSnap or BetterTouch, however seeing the same problem. Apps work fine with Chrome, but crashes on Brave. 

Thanks for looking into this @petemill !

**Comment by rebron** [comment]
cc: @jumde

**Comment by benlc** [comment]
Same here with BBT

**Comment by ron-wolf** [comment]
Same exact problem here. I can provide logs for several apps if helpful. The only one that still works for me is Google Keep.

**Comment by zhongke-chen** [comment]
Same issue with [Window Tidy](https://apps.apple.com/us/app/window-tidy/id456609775?mt=12)

**Comment by phao5814** [comment]
Thought I'd also mention that this issue also presents itself when using the [Alt Tab utility](https://github.com/lwouis/alt-tab-macos). I've also tried using the Nightly version of Brave and the same problem presents itself (Brave PWA will crash immediately upon launch).

@petemill just wondering if there are any updates on this issue?

**Comment by digitalmaster** [comment]
Yea, I think it might be with apps that require accessibility permissions. Started happening for me with a few other apps. Unforunately, as much as loved this app I was forced to switch back to Chrome w/Adblock because of this issue (plus some more rare slowdowns/crashes)

**Comment by iefremov** [comment]
Reproduced easily:
1) install BetterTouchTool and grant it a privacy permission in Accessibility section
2) install starbucks PWA
3) Launch PWA and try moving the window

**Comment by dsabanin** [comment]
Same problem with Timing app. This is a big deal. Please prioritize this.

**Comment by mdbraber** [comment]
Probably @bsclifton or @Brave-Matt could bump the  priority on this

**Comment by iefremov** [comment]
@rebron @bsclifton I'm working on this

**Comment by rightisleft** [comment]
So excited. Once this ticket is approved I can finally delete chrome!

**Comment by ctwhome** [comment]
Just to notice, also happening with [Cinch](https://www.irradiatedsoftware.com/cinch/)

**Comment by phao5814** [comment]
> Just to notice, also happening with [Cinch](https://www.irradiatedsoftware.com/cinch/).
> 
> It works with [Tiles](https://freemacsoft.net/tiles/) tho, in case you wanna check it out, and it's free.

@ctwhome FYI, if Cinch is the only app you use that causes Brave PWAs to crash, you might want to consider an alternative window manager like [Rectangle](https://rectangleapp.com/) (I have zero affiliation with Rectangle btw). As far as I know, Rectangle doesn't cause Brave PWAs to crash unlike the other window managers like BetterSnapTool etc.

**Comment by dennismkerr** [comment]
@phao5814 Thank you for that suggestion!  Since Google Chat is switching to PWA only, this was the last thing preventing me from fully moving from Chrome.

**Comment by nabn** [comment]
@phao5814 I use Rectangle, and it doesn't work for me if "Open as window" option is checked.

**Comment by phao5814** [comment]
> @phao5814 I use Rectangle, and it doesn't work for me if "Open as window" option is checked.

Hmmm strange. I have a feeling you might have another app / extension you use that is interfering with Brave. For me, [AltTab](https://alt-tab-macos.netlify.app/) is the thing that interferes. I just tried quitting AltTab and leaving Rectangle open and my Brave browser apps which "Open as window" opened with no issues.

Btw my version of Rectangle is Version 0.44 (49).

Try disabling some other apps that require MacOS Accessibility permissions to find the culprit.

**Comment by nabn** [comment]
Thanks @phao5814 !

Can confirm that quitting AltTab fixed the problem. Opening AltTab again makes already open brave shortcut immediately crash.

Checking the Console app, I can see a few crashlogs under "Logs" which are in the lines of..

```Class MenuControllerCocoa is implemented in both /Users/nabn/Applications/Brave Browser Apps.localized/Google Chat.app/Contents/MacOS/app_mode_loader (0x1035eb6a8) and /Applications/Brave Browser.app/Contents/Frameworks/Brave Browser Framework.framework/Versions/89.1.21.77/Brave Browser Framework (0x11a2673e0). One of the two will be used. Which one is undefined```

Followed by
`Service exited due to SIGABRT`

**Comment by m1guelpf** [comment]
Any updates on this?

**Comment by ron-wolf** [comment]
I think this has been fixed for me at some point in the past few months; not sure what did it. Will test with some other apps soon so I can tell for sure.

**Comment by m1guelpf** [comment]
@ron-wolf It's still happening for me with the latest production build 😅

**Comment by wknapik** [comment]
I hear https://community.brave.com/t/bug-report-macos-accessibility-immediately-causes-brave-web-apps-to-close/156666 is also caused by this.

**Comment by wScottSh** [comment]
One more shoutout that I'm running into this problem with BetterSnapTool. I see it's recently been escalated to p1, which is cool.

**Comment by geekysrm** [comment]
Still happening for me on the latest prod build.

**Comment by Brave-Matt** [comment]
+1 Community:
https://community.brave.com/t/create-shortcut-open-as-window-apps-wont-open-in-brave-mac-os/257346/6

**Comment by jorisc90** [comment]
Just wanted to add the application "[Witch](https://manytricks.com/witch/)" to the list of applications that cause this issue. I had hopes that it would circumvent the issue by being a PrefPane application, but it still has a daemon running that requires the Accessibility APIs.

**Comment by bsclifton** [comment]
Hi folks - this may be fixed with Chromium 92 which will be in product version 1.27 which we're aiming to release soon. 

When this new version pops up, please let us know if the issue is resolved 😄  If you're using Beta or Nightly, you should already have the update

**Comment by jorisc90** [comment]
I can confirm this issue has been resolved for me in version 1.27.

**Comment by dell-mic** [comment]
Can confirm my issue related to better touch tool functionality seems to be resolved as well!

**Comment by bsclifton** [comment]
Great, thanks for confirming folks! I'll go ahead and close this issue 😄

**Comment by maxhoheiser** [comment]
sadly I still run into that issue all the time, even with bettertouchtool & bettersnapttool dsiabled - after each update, the standalone apps open in a blank tab and dont load. the only thing that works for me is:
1. create a new brave profile
2. quite brave
3. reopen brave selecting the new profile + opening any standalone app
4. quitting it and deleting the new profile
 
but this is annoying :(

**Comment by mjs2430** [comment]
I'm still having this issue, with no success following any of the fixes above. I'm on [Version 1.35.104 Chromium: 98.0.4758.109 (Official Build) (arm64)]

Any open with window shortcut I create just immediately crashes, no window ever appears. The app icon in MacOS just blinks when you double-click on it.

**Comment by craigmdennis** [comment]
Updated to V 1.35.104 and now experiencing this with all PWAs

**Comment by martin-braun** [comment]
@craigmdennis Please make a new issue, the error is way too generic, it probably has a different stack trace.

**Comment by craigmdennis** [comment]
I have this: https://community.brave.com/t/updating-to-1-35-104-breaks-all-pwas/346680 but have no idea how to debug because no window even opens.

**Comment by mjs2430** [comment]
After installing and attempting to launch a PWA, I get this in my MacOS terminal

`2022-02-24 12:41:00.601 Brave Browser[68264:3709698] AppShim: Connection received from pid 70021
2022-02-24 12:41:00.607 Brave Browser[68264:3698101] AppShim: Received OnShimConnected from pid 70021
[68264:259:0224/124100.607252:ERROR:app_shim_manager_mac.cc(708)] The attaching app shim's code signature is invalid.
2022-02-24 12:41:00.607 Brave Browser[68264:3698101] AppShim: Performing OnFailedToConnectToHost result 6 for pid 70021
2022-02-24 12:41:00.607 Brave Browser[68264:3698101] AppShim: Closing pid 70021
[68264:259:0224/124100.770338:ERROR:web_app_shortcut_mac.mm(143)] Application has already been terminated.
2022-02-24 12:41:01.275 Brave Browser[68264:3709404] AppShim: Connection received from pid 70025
2022-02-24 12:41:01.281 Brave Browser[68264:3698101] AppShim: Received OnShimConnected from pid 70025
[68264:259:0224/124101.281119:ERROR:app_shim_manager_mac.cc(708)] The attaching app shim's code signature is invalid.
2022-02-24 12:41:01.281 Brave Browser[68264:3698101] AppShim: Performing OnFailedToConnectToHost result 6 for pid 70025
2022-02-24 12:41:01.281 Brave Browser[68264:3698101] AppShim: Closing pid 70025
2022-02-24 12:41:03.935 Brave Browser[68264:3709698] AppShim: Connection received from pid 70041
2022-02-24 12:41:03.943 Brave Browser[68264:3698101] AppShim: Received OnShimConnected from pid 70041
[68264:259:0224/124103.959274:ERROR:app_shim_manager_mac.cc(708)] The attaching app shim's code signature is invalid.
2022-02-24 12:41:03.959 Brave Browser[68264:3698101] AppShim: Performing OnFailedToConnectToHost result 6 for pid 70041
2022-02-24 12:41:03.959 Brave Browser[68264:3698101] AppShim: Closing pid 70041
[68264:259:0224/124103.962018:ERROR:CONSOLE(1)] "[Shields]: Can't request shields panel data for tabId: 13.  Error: No tab url specified", source: chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/out/brave_extension_background.bundle.js (1)
[68264:259:0224/124103.962356:ERROR:CONSOLE(1)] "[Shields]: Can't request shields panel data for tabId: 13.  Error: No tab url specified", source: chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/out/brave_extension_background.bundle.js (1)
2022-02-24 12:41:04.232 Brave Browser[68264:3709698] AppShim: Connection received from pid 70045
2022-02-24 12:41:04.238 Brave Browser[68264:3698101] AppShim: Received OnShimConnected from pid 70045
[68264:259:0224/124104.238702:ERROR:app_shim_manager_mac.cc(708)] The attaching app shim's code signature is invalid.
2022-02-24 12:41:04.238 Brave Browser[68264:3698101] AppShim: Performing OnFailedToConnectToHost result 6 for pid 70045
2022-02-24 12:41:04.238 Brave Browser[68264:3698101] AppShim: Closing pid 70045
[68264:259:0224/124104.322899:ERROR:web_app_shortcut_mac.mm(143)] Application has already been terminated.
2022-02-24 12:41:04.701 Brave Browser[68264:3709698] AppShim: Connection received from pid 70048
2022-02-24 12:41:04.707 Brave Browser[68264:3698101] AppShim: Received OnShimConnected from pid 70048
[68264:259:0224/124104.707494:ERROR:app_shim_manager_mac.cc(708)] The attaching app shim's code signature is invalid.
2022-02-24 12:41:04.707 Brave Browser[68264:3698101] AppShim: Performing OnFailedToConnectToHost result 6 for pid 70048
2022-02-24 12:41:04.707 Brave Browser[68264:3698101] AppShim: Closing pid 70048
`

**Comment by craigmdennis** [comment]
This is from the Mac Console when opening the install dialog, installing, and trying to open the installed app:

```
error	17:45:39.637301+0000	kernel	Sandbox: mdworker_shared(7777) deny(1) file-read-data /Users/craigmdennis/Applications/Brave Browser Apps.localized/.localized ()
error	17:45:39.642031+0000	mdworker_shared	Error (non-fatal) enumerating <private>: Error Domain=NSCocoaErrorDomain Code=257 "The file “.localized” couldn’t be opened because you don’t have permission to view it." UserInfo={NSURL=file:///Users/craigmdennis/Applications/Brave%20Browser%20Apps.localized/.localized, NSFilePath=/Users/craigmdennis/Applications/Brave Browser Apps.localized/.localized, NSUnderlyingError=0x157724780 {Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted"}}
error	17:45:39.703677+0000	mdworker_shared	Error (non-fatal) enumerating <private>: Error Domain=NSCocoaErrorDomain Code=257 "The file “.localized” couldn’t be opened because you don’t have permission to view it." UserInfo={NSURL=file:///Users/craigmdennis/Applications/Brave%20Browser%20Apps.localized/.localized, NSFilePath=/Users/craigmdennis/Applications/Brave Browser Apps.localized/.localized, NSUnderlyingError=0x152e21820 {Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted"}}
error	17:45:40.179830+0000	tccd	failed to get text offset for /Users/craigmdennis/Applications/Brave Browser Apps.localized/Twitter.app/Contents/MacOS/app_mode_loader[7778]: (#3) No such process
error	17:45:40.687342+0000	kernel	1 duplicate report for Sandbox: mdworker_shared(7777) deny(1) file-read-data /Users/craigmdennis/Applications/Brave Browser Apps.localized/.localized ()
error	17:45:40.758764+0000	tccd	failed to get text offset for /Users/craigmdennis/Applications/Brave Browser Apps.localized/Twitter.app/Contents/MacOS/app_mode_loader[7783]: (#3) No such process
error	17:45:43.861502+0000	tccd	Prompting policy for hardened runtime; service: kTCCServiceAppleEvents requires entitlement com.apple.security.automation.apple-events but it is missing for accessing={<TCCDProcess: identifier=app_mode_loader, pid=7787, auid=501, euid=501, binary_path=/Users/craigmdennis/Applications/Brave Browser Apps.localized/Twitter.app/Contents/MacOS/app_mode_loader>}, requesting={<TCCDProcess: identifier=com.apple.appleeventsd, pid=505, auid=55, euid=55, binary_path=/System/Library/CoreServices/appleeventsd>},
error	17:45:43.994489+0000	runningboardd	RBSStateCapture remove item called for untracked item 396-375-4410 (target:[app<application.com.brave.Browser.app.jgeocpdicgmkeemopbanhokmhcgcflmi.8979265.8979274(501)>:7787])
error	17:45:43.994535+0000	runningboardd	RBSStateCapture remove item called for untracked item 396-346-4411 (target:[app<application.com.brave.Browser.app.jgeocpdicgmkeemopbanhokmhcgcflmi.8979265.8979274(501)>:7787])
error	17:45:43.994573+0000	runningboardd	RBSStateCapture remove item called for untracked item 396-346-4409 (target:[app<application.com.brave.Browser.app.jgeocpdicgmkeemopbanhokmhcgcflmi.8979265.8979274(501)>:7787])
error	17:45:44.003897+0000	kernel	Sandbox: gamed(2853) deny(1) file-read-data /Users/craigmdennis/Applications/Brave Browser Apps.localized/Twitter.app
error	17:45:44.286789+0000	tccd	failed to get text offset for /Users/craigmdennis/Applications/Brave Browser Apps.localized/Twitter.app/Contents/MacOS/app_mode_loader[7793]: (#3) No such process
error	17:45:44.288223+0000	runningboardd	RBSStateCapture remove item called for untracked item 396-346-4417 (target:[app<application.com.brave.Browser.app.jgeocpdicgmkeemopbanhokmhcgcflmi.8979265.8979274(501)>:7793])
error	17:45:44.288264+0000	runningboardd	RBSStateCapture remove item called for untracked item 396-568-4415 (target:app<application.com.brave.Browser.app.jgeocpdicgmkeemopbanhokmhcgcflmi.8979265.8979274(501)>)
error	17:45:46.067034+0000	tccd	failed to get text offset for /Users/craigmdennis/Applications/Brave Browser Apps.localized/Twitter.app/Contents/MacOS/app_mode_loader[7805]: (#3) No such process
```

**Comment by craigmdennis** [comment]
I have created a new issue: #21302

**Comment by martin-braun** [comment]
@craigmdennis that's the way to go 👍🏻. Now it gets the attention is needs.
